packages/field/src/components/time/index.tsx

```
@@ -0,0 +1,35 @@
+          return (
+            <DatePicker
+              v-model:value={formState![column.value?.dataIndex as string]}
+              placeholder={`请输入${column.value?.title}`}
+              allowClear
+              valueFormat={VALUE_FORMAT}
+              showTime
+              {...column.value?.fieldProps}
+            />
+          );
+        case "read":
+        default:
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)